torch/_dynamo/variables/torch.py

```diff
@@ -1033,14 +1034,17 @@ def call_function(
             # the spec not a graphable type, so we still have to reconstruct it
             # into a python object, and store it as a constant attribute on the
             # fx graph.
-            #
-            # TODO handle `pytree._register_constant`-ed values.
             try:
                 input_spec = input_spec_vt.as_python_constant()
             except NotImplementedError:
                 unimplemented(
                     """
-This error is most likely due to a call to `nonstrict_trace`-ed function, where one of the argument contains object of a type that has been (or needs to be) `torch.utils._pytree.register_constant`-ed. We currently don't support that.
+TODO werid case, a user registered type whose `pytree_flatten` that puts some
+non-reconstructible object into the context (which then got stored into the spec).
```

Comment:
do we have a good sense of what objects are reconstructible and what aren't, if we end up doing reconstruction via pycodegen?

Reply:
Not in general... So we have `as_python_constant`-able VTs, which is a subset of `reconstruct`-able VTs, which is a subset of all VTs.